app/components/new-osf-navbar/template.hbs

```diff
@@ -1,84 +1,132 @@
-<div class="osf-nav-wrapper">
-    <nav class="navbar navbar-inverse navbar-fixed-top" id="navbarScope" role="navigation">
-        <div class="container">
-            <div class="navbar-header">
-                {{!TOGGLE NAVIGATION BUTTON - XS SCREEN}}
-                <a type="button" role="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#secondary-navigation" onclick={{action 'closeSearch'}} aria-label={{t 'eosf.navbar.toggleSecondary'}}>
-                    <span class="icon-bar"></span>
-                    <span class="icon-bar"></span>
-                    <span class="icon-bar"></span>
-                </a>
-                {{!OSF BRAND}}
-                <a class="navbar-brand" href="/" aria-label={{t 'eosf.navbar.goHome'}}>
-                    <span class="osf-navbar-logo"></span>
-                </a>
+<div class="nav-item-group global-nav">
+
+    {{!--
+        OSF Branding
+    --}}
+    {{#link-to "index"}}
+        <span class="navbar-logo"></span>
+    {{/link-to}}
+    {{!--
+        NAME OF CURRENT SERVICE - HOME, PREPRINTS, REGISTRIES, or MEETINGS
+    --}}
+    {{#link-to "index" class="service-link"}}
+        <span class="hidden-xs"> OSF </span>
+    {{/link-to}}
+
+    {{!--
+        Primary Navigation - Toggles between Home, Preprints, Registries, and Meetings
+    <div class="dropdown">
+
+        <button
```

Comment:
Does this commented out code need to be here still?

Reply:
This is the navbar dropdown